dev/com.ibm.ws.app.manager.springboot/resources/com/ibm/ws/app/manager/springboot/internal/resources/Messages.nlsprops

```diff
@@ -35,17 +35,17 @@ warning.could.not.thin.application=CWWKC0251W: Could not convert the application
 
-error.need.springboot.version.15=CWWKC0252E: Could not run the application because the wrong version of the Spring Boot feature is configured in server.xml. The application requires the feature springBoot-1.5 to be configured.
-error.need.springboot.version.15.explanation=The application requires Spring Boot version 1.5, but a wrong version of the Spring Boot feature is configured in the server.xml.  
-error.need.springboot.version.15.useraction=The springBoot-1.5 feature needs to be configured in the server.xml.
+error.need.springboot.version.15=CWWKC0252E: The application failed to start because the wrong version of the Spring Boot feature is configured in the server.xml file. The application requires the feature springBoot-1.5 to be configured.
+error.need.springboot.version.15.explanation=The application requires Spring Boot version 1.5, but a wrong version of the Spring Boot feature is configured in the server.xml file.  
+error.need.springboot.version.15.useraction=Configure the springBoot-1.5 feature in the server.xml file.
 
-error.need.springboot.version.20=CWWKC0253E: Could not run the application because the wrong version of the Spring Boot feature is configured in server.xml. The application requires the feature springBoot-2.0 to be configured.
-error.need.springboot.version.20.explanation=The application requires Spring Boot version 2.0, but a wrong version of the Spring Boot feature is configured in the server.xml.  
-error.need.springboot.version.20.useraction=The springBoot-2.0 feature needs to be configured in the server.xml.
+error.need.springboot.version.20=CWWKC0253E: The application failed to start because the wrong version of the Spring Boot feature is configured in the server.xml file. The application requires the feature springBoot-2.0 to be configured.
+error.need.springboot.version.20.explanation=The application requires Spring Boot version 2.0, but a wrong version of the Spring Boot feature is configured in the server.xml file.  
+error.need.springboot.version.20.useraction=Configure the springBoot-2.0 feature in the server.xml file.
 
-error.missing.servlet=CWWKC0254E: Could not run the application because it requires a Servlet feature to be configured in the server.xml.
-error.missing.servlet.explanation=The application is a web application and it requires a Servlet feature to be configured in the server.xml.
-error.missing.servlet.useraction=A Servlet feature needs to be configured in the server.xml.
+error.missing.servlet=CWWKC0254E: The application failed to start because it requires a Servlet feature to be configured in the server.xml file.
+error.missing.servlet.explanation=The application is a web application, and it requires a Servlet feature to be configured in the server.xml file.
+error.missing.servlet.useraction=Configure a Servlet feature in the server.xml file.
```

Comment:
"Servlet" doesn't need to be capitalized unless we're talking about a specific one, such as "Servlet 4.0."